packages/lib-classifier/src/store/feedback/helpers/metadata-to-rules.js

```diff
@@ -7,6 +7,9 @@ function metadataToRules (metadata = {}) {
     const stringValue = value.toString()
 
     if (prefix === '#feedback' && stringValue) {
+      if (isNaN(ruleIndex)) {
+        console.error(`Subject metadata feedback rule index ${ruleIndex} is improperly formatted. The feedback rule index should be an integer.`)
```

Comment:
Do these errors get logged anywhere other than the console? If feedback isn't set up properly, how do we find out?

Reply:
There are some [basic validations in the Project Builder](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/features/feedback/lab/helpers/default-validations.js) but nothing that checks the rule index.